lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2729,9 +2729,7 @@ public function get_root_layout_rules( $selector, $block_metadata ) {
 		$css .= '.wp-site-blocks > .alignright { float: right; margin-left: 2em; }';
 		$css .= '.wp-site-blocks > .aligncenter { justify-content: center; margin-left: auto; margin-right: auto; }';
 
-		$block_gap_value       = $this->theme_json['styles']['spacing']['blockGap'] ?? '0.5em';
```

Comment:
I'm still seeing a fallback in the JS

https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/global-styles/use-global-styles-output.js#L479-480

Should the fallback of `'0.5em'` be used in the if statement if `static::get_property_value` returns empty?

Reply:
Good question! A couple of considerations:

* Core's `lib/theme.json` includes a `styles.spacing.blockGap` value of `24px`, so in practice, was the `0.5em` value ever being output? I'm curious if the `0.5em` default spacing was ever being displayed visually for anyone 🤔
* I think the `0.5em` fallback was originally intended for flex layouts so that on classic themes where there might not be any blockGap value, things like Buttons spacing would still get a visually pleasing default value. In the case of root layout rules, that might not be necessary?